### PR TITLE
feature(k8s): add doc for grafana

### PR DESCRIPTION
**Pull request by wilfriedroset**
Signed-off-by: Wilfried Roset <wilfriedroset@users.noreply.github.com>

### CODE REVIEW

**Review by OVH-Team-Guides — APPROVED** [review]

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image img_2780.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Empty alt attribute for image img_2782.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Empty alt attribute for image img_2784.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Empty alt attribute for image img_2786.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Other images have empty alt attributes
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/platform/kubernetes-k8s/installing-grafana/guide.en-gb.md
  | ScannerError: while scanning a simple key
  in "<unicode string>", line 6, column 1:
    <style>
    ^
could not find expected ':'
  in "<unicode string>", line 8, column 15:
         font-size: 14px;
                  ^
CRITICAL: RuntimeError: Error encountered
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image img_2780.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Empty alt attribute for image img_2782.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Empty alt attribute for image img_2784.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Empty alt attribute for image img_2786.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/billing/loyalty_account/guide.es-us.md
WARNING: Other images have empty alt attributes
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/platform/kubernetes-k8s/installing-grafana/guide.en-gb.md
  | ScannerError: while scanning a simple key
  in "<unicode string>", line 6, column 1:
    <style>
    ^
could not find expected ':'
  in "<unicode string>", line 8, column 15:
         font-size: 14px;
                  ^
CRITICAL: RuntimeError: Error encountered
```

**Review by OVH-Team-Guides — APPROVED** [review]

File: pages/platform/kubernetes-k8s/installing-grafana/guide.en-asia.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+> [!warn]
+> Don't forget to update `[YOUR_INGRESS_URL]`
+
+Save the content above in a file name `values.yml` as we will use it in the next step.
+
+### Deploying the pod
+
+With all the settings define in the previous step, we can install Grafana using the following command:
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/installing-grafana/guide.en-asia.md:91]:
```suggestion
With all the settings defined in the previous step, we can install Grafana using the following command:
```

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @wilfriedroset, deployment will happen very soon

File: pages/platform/kubernetes-k8s/installing-grafana/guide.en-asia.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+```bash
+kubectl get secret --namespace default grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+## Cleaning up
+
+To clean up your cluster, simply use Helm to delete your Grafana release.
+
+```bash
+helm delete grafana
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/installing-grafana/guide.en-asia.md:122]:
```suggestion
helm uninstall grafana
```

**Review by XavierDuthil — CHANGES_REQUESTED** [review]

**Pull request merged**